integration-tests/src/test/java/org/zowe/apiml/functional/gateway/InMemoryRateLimiterFilterFactoryIntegrationTest.java

```diff
@@ -31,7 +31,7 @@ public class InMemoryRateLimiterFilterFactoryIntegrationTest {
 
     private static WebTestClient client;
 
-    final int bucketCapacity = 60;
+    final int bucketCapacity = 20;
```

Comment:
I think this number was specified for HA integration tests, could you verify this branch with the ITs pipeline?
If it needs to be updated we should have it parameterized depending on whether it's run as HA or single instance

Reply:
Yep, in our HA ITs it was failing, because the calls are routed  randomly to different GW instances and the rate limit is set on instance level so it might never reach this limit.

Reply:
It's now configurable via "gateway.bucketCapacity." I'm not sure if that would solve your problem with HA, as it can still be random, and you will not get the exact number of requests for each instance. It was probably passing due to missing the lower boundary check.